JWBestPracticeApps/FeedTableViewController/FeedTableViewController/Feed/FeedViewModel.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  FeedViewModel.swift
+//  FeedTableViewControllerV4
+//
+//  Created by Amitai Blickstein on 5/2/22.
+//
+
+import Foundation
+import JWPlayerKit
+
+protocol FeedViewModelDelegate: AnyObject {
+    func onFetchCompleted(with newIndexPathsToReload: [IndexPath]?)
+}
+
+final class FeedViewModel {
```

Comment:
Because this is for outside developers, served as a tutorial of sorts, add header doc for all properties and classes and protocols. Think of this as a teaching module. The more clear what the properties and classes are, and what their purpose is, the easier it'll be for others to understand and modify.

For example, what is `total`? Does it need to be an optional? Looking at this, to me, I understand this as the number of items within the `items` array, but if that's the case, it makes no sense since I could simply call `items.count`

Reply:
Removed this entirely in the latest version.